### PR TITLE
auto some egressfirewall cases

### PR DESCRIPTION
**Pull request by huiran0826**
Auto some egressfirewall cases. Here is the logs.

https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2177/console
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2176/console
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2175/console
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2174/console

Can you help review? Thanks!
@zhaozhanqi @weliang1 @anuragthehatter @rbbratta

### CODE REVIEW

File: features/networking/egress-ingress.feature [hunk 1]
```diff
@@ -354,3 +354,114 @@ Feature: Egress-ingress related networking scenarios
...
+      | curl | -k | --connect-timeout | 5 | --head | https://<%= cb.yahoo_ip %>:443 |
+    Then the step should succeed
+    When I execute on the pod:
+      | curl | --connect-timeout | 5 | --head | <%= cb.google_ip %>:80 |
+    Then the step should succeed
+    When I execute on the pod:
+      | curl | -k | --connect-timeout | 5 | --head | https://<%= cb.google_ip %>:443 |
+    Then the step should fail
+    When I execute on the pod:
+       | curl | --connect-timeout | 5 | --head | www.test.com |
```
Comment (liangxia) [inline features/networking/egress-ingress.feature:389]: Indention issue.

File: features/networking/egress-ingress.feature [hunk 2]
```diff
@@ -354,3 +354,114 @@ Feature: Egress-ingress related networking scenarios
...
+    And I run the :create admin command with:
+      | f | limit_policy.json   |
+      | n | <%= project.name %> |
+
+    # Create svc/pods
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given 1 pods become ready with labels:
```
Comment (liangxia) [inline features/networking/egress-ingress.feature:427]: s/1 pods/1 pod/

Comment (huiran0826) [reply features/networking/egress-ingress.feature:427]: Thanks for checking, but the step definition here is 'pods', not 'pod'

File: features/networking/egress-ingress.feature (hunk 1 above)
Comment (huiran0826) [reply features/networking/egress-ingress.feature]: Fixed it, thanks!

File: testdata/networking/ovn-egressfirewall/egressfirewall-policy2.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+kind: EgressFirewall
+apiVersion: k8s.ovn.org/v1
+metadata:
+  name: default
+  namespace: test1
```
Comment (rbbratta) [inline testdata/networking/ovn-egressfirewall/egressfirewall-policy2.yaml:5]: `namespace` is empty for the other policies, should we remove it here?

File: features/networking/egress-ingress.feature (hunk 2 above)
Comment (liangxia) [reply features/networking/egress-ingress.feature:427]: Fixed in https://github.com/openshift/verification-tests/pull/1532/files

Comment (liangxia) [reply features/networking/egress-ingress.feature:427]: Could you update here ?

File: testdata/networking/ovn-egressfirewall/egressfirewall-policy2.yaml (hunk 3 above)
Comment (huiran0826) [reply testdata/networking/ovn-egressfirewall/egressfirewall-policy2.yaml]: Sure, removed it.

File: features/networking/egress-ingress.feature (hunk 2 above)
Comment (huiran0826) [reply features/networking/egress-ingress.feature]: Thanks @liangxia , updated it.

**Pull request merged**